### ISSUE TITLE
Centralisation as emphasis and In-line graphics

### ISSUE THREAD

**Issue by m-k94** [DCMLab/ddd#29]
I'm just looking at different (unprocessed) documents to see if any are as broken as Weitzmann 1861 or have any interesting passages for segmentation.

This might be one:

![image](https://user-images.githubusercontent.com/75686579/105062254-b25ecf80-5a7a-11eb-8d87-c0ca8013ed0e.png)

Firstly, it's not clear whether the "**Centrumston oder phonische Mittelpunkt**..." line is supposed to be an actual heading, as it's embedded in the same sentence as what comes before and after it. I thought of it as an added emphasis, but for that, being centralized on the page is very relevant. Is there a way to represent that using Transkribus?

If it's marked as "heading", I suppose it would break up the sentence. Also, it seems like the author uses numbered headings (roman numerals and capital letters), so it might be best to actually interpret the whole thing as being the same paragraph?

Secondly, in the consequent paragraph you see a different, slightly related issue: There are little pictures inline with the text. You could easily merge the "text" baselines I suppose, but then my question would be how to know later where the graphics are supposed to go? Also has to be understood that the first image relates to the upper, and the second one to the lower line, despite being on the same level.

**Comment by fabianmoss** [comment]
This is a bit weird indeed. I think it is meant as a very strong emphasis and not as a structural marker. I agree that it should be just one paragraph. Maybe include in an extra comment describing the centering. But we wouldn't lose anything important if we just display it boldface.

Regarding the second issue (inline scores), I will think about a solution and come back to it.

**Comment by m-k94** [comment]
The text flow is rather unusual: it actually goes down in between the musical examples, and then the bit on the right is read down at well. I solved it by turning the text-bits between the examples into individual "lines" and making sure that the reading order is correct.

![image](https://user-images.githubusercontent.com/75686579/106328937-e8713000-6280-11eb-8e95-2bb8059e7276.png)

Is that acceptable, as a solution? I didn't really want to break up the paragraph as a unit. It's a rare case, but might happen sometimes.

**Comment by fabianmoss** [comment]
Yes this solution is good! Since we will merge all lines within a paragraph later anyways, having the in-line musical examples as you did it will give the desired output.
